recipes/libvalkey/recipe.yaml

```diff
@@ -0,0 +1,74 @@
+schema_version: 1
+
+context:
+  name: libvalkey
+  version: "0.1.0"
+
+package:
+  name: "${{ name }}"
+  version: "${{ version }}"
+
+source:
+  url: https://github.com/valkey-io/libvalkey/archive/refs/tags/${{ version }}.zip
+  sha256: "118e4cf50daf32645d267740f431e81c91bc85f332ea14a8051ddab59bba79c5"
+  patches:
+    # remove this after https://github.com/valkey-io/libvalkey/pull/203
+    - patches/0000-tls.patch
+
+build:
+  number: 0
+  script:
+    content:
+      - if: linux
+        then: cmake . -B build $CMAKE_ARGS  -D ENABLE_TLS=1 -D ENABLE_RDMA=1 -G Ninja
+      - if: osx
+        then: cmake . -B build $CMAKE_ARGS  -D ENABLE_TLS=1 -G Ninja
+      - if: win
+        then: cmake . -B build %CMAKE_ARGS% -D ENABLE_TLS=1 -G Ninja
+
+      - ninja -v -C build install
+
+requirements:
+  run_exports:
+    - ${{ pin_subpackage('libvalkey', upper_bound='x.x.x') }}
+
+  build:
+    - "${{ compiler('c') }}"
+    - "${{ stdlib('c') }}"
+    - if: unix
+      then: make
+    - cmake
+    - ninja
+    - pkg-config
+
+  host:
+    - openssl >=3
+    - glib >=2,<3a0
```

Comment:
Are you statically linking to glib? rattler doesn't detect any links at runtime.

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1245276&view=logs&j=4cabe686-70ae-553a-7fd0-310379f2cbac&t=6a4fc7c9-c31a-5115-eff9-6479d72b69ff&l=1020

Reply:
OK, glib is only used for testing. I removed it here.